docs/make.jl

```diff
@@ -183,23 +196,39 @@ function add_badges(badges)
 end
 
 # Run Literate on them all
+@debug "Processing literate files"
 for file in literate_files
+    @debug "Processing" file.src file.dest
+    # Skip all flies that did not change since last build
+    if mtime(file.src) <= mtime(file.dest) && !DEBUG
+        @debug "Skipping up-to-date file"
+        continue
+    end
```

Comment:
Could handle debug printing more elegantly here by printing a different message depending on the `mtime` check.